src/xmlnode.h

```diff
@@ -116,13 +116,13 @@ class XmlNode
     bool isCommentNode();
```

Comment:
Replace `isTextNode()` with `isText()` and `isCommentNode()` with `isComment()`.

Reply:
Given these are actually checking the type of the XML node, I think it is more correct to leave them as `isTextNode()` and `isCommentNode()`. Ideally we would also be checking in `isElement()` that we actually have an XML Element rather than simply grabbing the name of the node.

Reply:
Ok, then I would still have `isText()` and `isComment()` and then have `isElement()` check that we actually have an XML element.